pilot/pkg/model/endpointshards_test.go

```diff
@@ -25,8 +25,8 @@ func TestUpdateServiceAccount(t *testing.T) {
 	)
 
 	cluster1Endppoints := []*IstioEndpoint{
-		{Address: "10.172.0.1", ServiceAccount: "sa1"},
```

Comment:
For many of these unit tests can you add some dual-stack variants?

```golang
  {Addresses: []string{"10.172.0.1", "fd08::3f:01"}, ServiceAccount: "sa1"},
```

and also a test for 

```golang
  {Addresses: []string{"fd08::3f:01","10.172.0.1"}, ServiceAccount: "sa1"},
```

Reply:
Hi @jacob-delgado, sure, that's good idea. In fact I did some yesterday, but there are some errors as well. I believe that there should be some other changes for it, so have to complete the IstioEndpoint change at first. And thanks for your remind!

Reply:
Please add the tests as Jacob has mentioned.